src/fu-engine.c

```diff
@@ -2399,19 +2399,17 @@ fu_engine_get_plugins (FuEngine *self)
 }
 
 /**
- * fu_engine_get_device_by_id:
+ * fu_engine_get_device:
  * @self: A #FuEngine
- * @device_id: A string
- * @error: A #GError or NULL
- *
- * Returns the device from the engine or NULL if not found
+ * @device_id: A device ID
+ * @error: A #GError, or %NULL
  *
- * Returns: (transfer full): a #FuDevice
+ * Gets a specific device.
  *
- * Since: 1.5.0
+ * Returns: (transfer full): a device, or %NULL if not found
```

Comment:
This should still have the clause mentioning when it was introduced.  IE:
` * Since: 1.5.0	`

Reply:
This function has been there from before.... I just eliminated the _by_id that was going to be introduced on 1.5.0

Reply:
Oh I see... OK.
LGTM then